src/app/globals.css

```diff
@@ -23,3 +30,46 @@ body {
     text-wrap: balance;
   }
 }
+
+/* React Calendar heatmap*/
+.react-calendar-heatmap .color-myCal-0,
+.color-myCal-0 {
+  fill: rgb(var(--color-cal-0));
+}
+
+.react-calendar-heatmap .color-myCal-1,
+.color-myCal-1 {
+  fill: rgb(var(--color-cal-1));
+}
+.react-calendar-heatmap .color-myCal-2,
+.color-myCal-2 {
+  fill: rgb(var(--color-cal-2));
+}
+.react-calendar-heatmap .color-myCal-3,
+.color-myCal-3 {
+  fill: rgb(var(--color-cal-3));
+}
+.react-calendar-heatmap .color-myCal-4,
+.color-myCal-4 {
+  fill: rgb(var(--color-cal-4));
+}
+.react-calendar-heatmap .color-myCal-5,
+.color-myCal-5 {
+  fill: rgb(var(--color-cal-5));
+}
+
+.react-calendar-heatmap .react-calendar-heatmap-month-label,
+.react-calendar-heatmap .react-calendar-heatmap-weekday-label {
+  font-size: 10px;
+  line-height: 20px;
+  font-weight: 500;
+  fill: #55597d;
+}
+
+.react-calendar-heatmap .react-calendar-heatmap-weekday-label {
+  padding-right: 5px;
+}
+
+.react-calendar-heatmap-weekday-labels {
+  transform: translate(0px, 14px);
+}
```

Comment:
https://www.loom.com/share/82e91e675dde4bd5b1b61e59e71a8e48

Looks great already- don't think you need to change anything. But this is what we were talking about before with attempting to allow the 'overflow' on the svg itself.

Your solution may be best- as this feature technically doesn't have wide availability as a standard. I've only tested on three browsers- it's possible certain desktop or mobile browsers would just clip the text!

https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/overflow